### PR TITLE
Add instruction on how to test a pull request change

### PR DESCRIPTION
**Pull request by lemeurherve**

### CODE REVIEW

**Review by smerle33 — DISMISSED** [review]
🚀 
🙏

File: README.adoc [hunk 1]
```diff
@@ -208,4 +208,9 @@ buildDockerImage_k8s('plugins-site-api')
 * (Open)JDK v8
 * Maven 3.6.x
 
-===
+=== HowTo test a pull request change
+
+By adding `@Library('pipeline-library@pull/<your-pr-number>/head') _` at the top of a Jenkinsfile from a repository builded on one of the *.ci.jenkins.io instances, you can test in situ your pipeline library pull request.
```
Comment (MarkEWaite) [inline README.adoc:213]: Phrasing change for your consideration
```suggestion
By adding `@Library('pipeline-library@pull/<your-pr-number>/head') _` at the top of a Jenkinsfile from a repository built on one of the *.ci.jenkins.io instances, you can test your pipeline library pull request on ci.jenkins.io.
```

**Review by MarkEWaite — APPROVED** [review]
Two optional comments for your consideration.  Nice technique!

**Review by timja — APPROVED** [review]

**Review by MarkEWaite — APPROVED** [review]

**Comment by jglick** [comment]
Thanks, I was looking for something like this recently! https://github.com/jenkins-infra/pipeline-library/pull/705#issuecomment-1653807217

**Pull request merged**